src/entities/redemption-info.ts

```diff
@@ -0,0 +1,23 @@
+import type { CheckoutStatusResponse } from "../networking/responses/checkout-status-response";
+
+/**
+ * This object gives you access to all purchase redemption data.
```

Comment:
What do you mean by **all** purchase redemption data?
```suggestion
 * This object gives you access to the purchase redemption data.
```